DSE_6.8_Release_Notes.md

```diff
@@ -2,6 +2,32 @@
 DSE 6.8.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any.
 Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version.
 
+# Release notes for 6.8.41
+18 December 2023
+
+## Components versions for DSE 6.8.41
+ * Apache Solr™ 6.0.1.4.2964
+ * Apache Spark™ 2.4.0.29
+ * Apache TinkerPop™ 3.4.14-20231030-479dc6d7
+ * Apache Tomcat® 8.5.94&ast;
+ * DSE Java Driver 1.10.0-dse-20220616 (DSE *internal-only* version)
+ * Netty 4.1.100.1.dse
+ * Spark JobServer 0.8.0.54
+
+**NOTE**: above-listed DSE Java Driver is an _internal-version_ only.
+If you're developing applications, please refer to the [Java Driver documentation](https://docs.datastax.com/en/driver-matrix/doc/java-drivers.html) to choose an appropriate version.
+
+## 6.8.41 DSE Core
+* Fixed deadlock in indexes initialization that occurs when the same table has both a secondary index and a search index and the entries in `IndexInfo` table are missing. Deadlock resolved by marking the SOLR index as built in a different thread than the main DSE thread. (DSP-23828)
```

Comment:
should `as` in "SOLR index as build in a .." be removed ?

Reply:
> should as in "SOLR index as build in a .." be removed ?

I have the feeling the `as` word allows us to express better we are assigning a state to the operation.
Those release notes are double-checked by the documentation team before the release, so I figured that if they did not change it, I should leave this wording as it was.

> also, I think I would rephrase the entire sentence as  The deadlock is resolved by building the SOLR index in a different thread than the main DSE thread

That would change the semantics because even before this ticket the index is built in a separate thread but the problem was caused by marking it as built in the main thread.

Reply:
> Those release notes are double-checked by the documentation team before the release,

in this case forget my rephrasing 

> problem was caused by marking it as built in the main thread.

Ok. then the problem is that I didn't understand the change :D

Reply:
It might be possible to improve the wording a bit to make things a little bit clearer, but the current wording isn't wrong.

If I were to suggest what I think is the smallest possible change to clarify the wording, I think something like:
> Fixed deadlock in indexes initialization that occurs when the same table has both a secondary index and a search index and the entries in `IndexInfo` table are missing.
> _**The**_ ~~D~~**d**eadlock _**is**_ resolved by marking the SOLR index as built in a different thread than the main DSE thread.

Reply:
The addition of `The` indeed makes it better as a deadlock was introduced in the first sentence, in the second sentence we are referring to `the` deadlock.